android/src/main/java/com/apparence/camerawesome/CameraPicture.java

```diff
@@ -81,41 +81,61 @@ public CameraPicture(Context context, CameraPreview cameraPreview, CameraSession
      * @param width
      * @param height
      */
-    public void setSize(int width, int height) {
-        this.size = new Size(width, height);
+    public void setPhotoSize(int width, int height) {
+        this.photoSize = new Size(width, height);
```

Comment:
should photoSize and videoSize be unset when either of them is set? 

(i don't know the lifecycle of the CameraPicture)

Reply:
Each time a new CameraAwesome widget is loaded a new CameraPicture object is created so this won't cause issues.